source/libraries/ContractInterfaces.ts

```diff
@@ -35,7 +35,7 @@
             const from = sender || this.accountManager.defaultAddress;
             const data = encodeMethod(abi, parameters);
             // TODO: remove `gas` property once https://github.com/ethereumjs/testrpc/issues/411 is fixed
-            const gas = await this.connector.ethjsQuery.estimateGas(Object.assign({ to: this.address, from: from, data: data, gas: new BN(6500000) }, attachedEth ? { value: attachedEth } : {} ));
+            const gas = await this.connector.ethjsQuery.estimateGas(Object.assign({ to: this.address, from: from, data: data }, attachedEth ? { value: attachedEth } : {} ));
```

Comment:
Updated TestRPC which included a fix so attaching gas to estimate gas is no longer necessary.

Reply:
I just realized this is in `ContractInterfaces.ts`.  Need to change the template it seems...